internal/nginx/config/servers.go

```diff
@@ -314,3 +316,14 @@ func createDefaultRootLocation() http.Location {
 		Return: &http.Return{Code: http.StatusNotFound},
 	}
 }
+
+func formatPathType(pathType v1beta1.PathMatchType) string {
```

Comment:
I wonder if we need this function at all. what if we just use the value of path.PathType as it. We can be guarded against changes by still having unit tests? 🤔  
with regards to empty value, the data plane package will ensure PathType is always said. Currently, the validation in https://github.com/nginxinc/nginx-kubernetes-gateway/blob/f74c06ded5c3d647b0909fe542f3e70dbc96774c/internal/state/graph/httproute.go#L519-L529 forced the path type to be always set.

Reply:
The main reason I added this is to keep consistency in the letter case of the path name, e.g. `/test_PathPrefix_route0` versus `/test_prefix_route0`.